tests/unit/test_reporter.py

```diff
@@ -64,6 +66,7 @@ def context(self, mocked__session):
             "project_name": "",
             "results": fake_results,
             "session": mocked__session,
+            "scanapi_version": get_distribution("scanapi").version,
```

Comment:
```suggestion
            "scanapi_version": '2.3.0',
```

One minor suggestions: as we don't want to test any external function, we can change it for what we expect: a string with a version. For us is irrelevant if the method `get_distribution` is working fine (we expect external methods were already tested externally).